### PR TITLE
Add creating halls

### PR DESCRIPTION
**Pull request by lukasz-ligizynski**

### CODE REVIEW

File: app/controllers/halls_controller.rb [hunk 1]
```diff
@@ -6,7 +6,12 @@ def index
     render json: Halls::Representer.new(halls).basic
   end
 
+  def new
```
Comment (wmaciejak) [inline app/controllers/halls_controller.rb]: This should be action `create`, not `new`. Please read about difference [here](https://stackoverflow.com/a/2472489)

File: config/routes.rb [hunk 2]
```diff
@@ -6,4 +6,5 @@
   get '/seances', to: 'seances#index'
   get '/tickets', to: 'tickets#index'
   get '/reservations', to: 'reservations#index'
+  get '/create_hall', to: 'halls#new'
```
Comment (wmaciejak) [inline config/routes.rb]: https://github.com/lukasz-ligizynski/Cinema/pull/12#discussion_r804406703

File: app/controllers/halls_controller.rb (hunk 1 above)
Comment (lukasz-ligizynski) [reply app/controllers/halls_controller.rb]: Okey looking into the diffrene

File: config/routes.rb (hunk 2 above)
Comment (lukasz-ligizynski) [reply config/routes.rb]: Got it :D I will send a later new version of all routes

File: app/concepts/halls/repository.rb [hunk 3]
```diff
@@ -4,11 +4,16 @@ module Halls
   class Repository
     class << self
       delegate :find_all,
```
Comment (wmaciejak) [inline app/concepts/halls/repository.rb:6]: What's the reason behind this delegation?

File: app/concepts/halls/use_cases/create_seats.rb [hunk 4]
```diff
@@ -9,10 +9,10 @@ def initialize(params:)
         @params = params
       end
 
-      def call(rows, columns)
+      def call(params: @params)
```
Comment (wmaciejak) [inline app/concepts/halls/use_cases/create_seats.rb:12]: When you initialize with params, you don't have to second time pass those parameters. If you defined it like[ a instance variable](https://www.rubyguides.com/2019/07/ruby-instance-variables/) (`@` sign, for instance `@params`), you can just use it everywhere in class.

File: config/routes.rb [hunk 5]
```diff
@@ -6,4 +6,5 @@
   get '/seances', to: 'seances#index'
   get '/tickets', to: 'tickets#index'
   get '/reservations', to: 'reservations#index'
+  get '/create_hall', to: 'halls#create'
```
Comment (wmaciejak) [inline config/routes.rb]: [This](https://github.com/lukasz-ligizynski/Cinema/pull/12#discussion_r804406703) comment is relevant also here

Comment (wmaciejak) [reply config/routes.rb]: Nvm, I've realized that you fixed it in another PR

Comment (lukasz-ligizynski) [reply config/routes.rb]: Yeah, sorry about it. I think I made too many PRs and some changes are in one place, and some in other

**Pull request merged**